WireFoundation/Sources/WireFoundation/Documentation.docc/Documentation.md

```diff
@@ -1,9 +1,10 @@
 # ``WireFoundation``
 
-Provide core shared code.
+Provide core shared code which is absoluteley needed in almost all targets.
 
 ## Overview
 
-WireFoundation provides some core shared code.
+It's important to keep the code in this target minimal because it will increase
+the overall build time.
```

Comment:
I feel like the `Overview` documentation in general should be more thorough about what this package contains and what it is used for with more description, e. g:

_WireFoundation_

**Crypto** : Provides shared code for secure random byte generation operations and AES256 cryptographic operations.

**Design**: Provides shared code for common design elements.

{...}

_WireTesting_

Provides XCTest helper functions

Reply:
This would be nice, I agree. But also here I suggest to fix this when we agreed on the new workspace structure.